test/runtests.jl

```diff
@@ -20,7 +20,29 @@ cd(dirname(@__FILE__)) do
 
     @everywhere include("testdefs.jl")
 
-    reduce(propagate_errors, nothing, pmap(runtests, tests; err_retry=false, err_stop=true))
+    const max_worker_rss = parse(Int, get(ENV, "JULIA_TEST_MAXRSS_MB", "200"))
```

Comment:
appveyor is on worker 75 by the time it's done: https://ci.appveyor.com/project/StefanKarpinski/julia/build/1.0.10076/job/l8u2mel4teldcv6e

travis only gets to 36, but that's still a bit overboard